static/app/views/issueDetails/streamline/header.tsx

```diff
@@ -76,245 +77,191 @@ export default function StreamlinedGroupHeader({
+const StatTitle = styled(Link)`
+  display: block;
+  text-decoration: underline;
+  text-decoration-style: dotted;
   color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+  line-height: 1;
+  align-self: flex-end;
+  justify-self: flex-end;
 `;
 
 const StatCount = styled(Count)`
-  font-size: ${p => p.theme.fontSizeExtraLarge};
   display: block;
+  font-size: 20px;
   line-height: 1;
+  text-align: right;
 `;
 
-const TitleWrapper = styled('h3')`
-  font-size: 20px;
-  margin: 0;
-  padding-bottom: 2px;
+const Message = styled(Tooltip)<{hasMessage: boolean}>`
+  overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
-  overflow: hidden;
-  color: ${p => p.theme.headingColor};
-
-  & em {
-    font-weight: ${p => p.theme.fontWeightNormal};
-    color: ${p => p.theme.textColor};
-    font-size: ${p => p.theme.fontSizeLarge};
-  }
-`;
-
-const TitleHeading = styled('div')`
-  display: flex;
-  line-height: 2;
-  gap: ${space(1)};
-`;
-
-const StyledBreak = styled('hr')`
-  margin-top: ${space(1.5)};
-  margin-bottom: 0;
-  margin-right: 0;
-  border-color: ${p => p.theme.border};
-`;
-
-const MessageWrapper = styled('div')`
-  display: flex;
-  color: ${p => p.theme.gray300};
-  gap: ${space(1)};
+  color: ${p => p.theme.subText};
+  font-style: ${p => (p.hasMessage ? 'initial' : 'italic')};
 `;
 
-const InfoWrapper = styled('div')<{isResolvedOrIgnored: boolean}>`
+const ActionBar = styled('div')<{isComplete: boolean}>`
   display: flex;
   justify-content: space-between;
-  gap: ${space(1)};
-  background: ${p =>
-    p.isResolvedOrIgnored
-      ? `linear-gradient(to right, ${p.theme.background}, ${Color(p.theme.success).lighten(0.5).alpha(0.15).string()})`
-      : p.theme.background};
-  color: ${p => p.theme.gray300};
-  padding: ${space(0.5)} 24px;
-  margin-right: 0;
-  margin-left: 0;
-  flex-wrap: wrap;
-`;
-
-const SidebarWorkflowWrapper = styled('div')`
-  display: flex;
-  gap: ${space(0.5)};
-  align-items: center;
-`;
-
-const WorkflowWrapper = styled('div')`
-  display: flex;
-  column-gap: ${space(2)};
+  gap: ${space(1.5)};
   flex-wrap: wrap;
-`;
-
-const Wrapper = styled('div')`
-  display: flex;
-  align-items: center;
-  gap: ${space(0.5)};
-`;
-
-const Header = styled('div')`
-  background-color: ${p => p.theme.background};
-  display: flex;
-  flex-direction: column;
-  border-bottom: 1px solid ${p => p.theme.border};
-
-  > * {
-    margin-right: 24px;
-    margin-left: 24px;
+  padding: ${space(1)} 24px;
+  border-bottom: 1px solid ${p => p.theme.translucentBorder};
+  position: relative;
+  transition: background 0.3s ease-in-out;
+  background: ${p => (p.isComplete ? 'transparent' : p.theme.background)};
+  &:before {
+    z-index: -1;
+    position: absolute;
+    inset: 0;
+    content: '';
+    background: linear-gradient(
+      to right,
+      ${p => p.theme.background},
+      ${p => Color(p.theme.success).lighten(0.5).alpha(0.15).string()}
```

Comment:
someone suggested adding a different color for archived (blue was the suggestion), this might be a good time to try that out?

Reply:
good point, lemme try that out